arm/Microsoft.EventGrid/topics/readme.md

```diff
@@ -38,6 +38,42 @@ This module deploys Event Grid
 
 To use Private Endpoint the following dependencies must be deployed:
 
+- Destination subnet must be created with the following configuration option - `"privateEndpointNetworkPolicies": "Disabled"`.  Setting this option acknowledges that NSG rules are not applied to Private Endpoints (this capability is coming soon). A full example is available in the Virtual Network Module.
```

Comment:
Double up. Remove one of the privateEndpoints

Reply:
done